samples/matter/light_bulb/boards/nrf54l15pdk_nrf54l15_cpuapp_0_2_1_release.overlay

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
```

Comment:
Did you check whether the file without the `_release` suffix is enough? If I understand our new build system if there is no file with the `_release` suffix it should look for the base board file, so if the release and without release files are the same it should be enough to have the base one only.

Reply:
It used not to work that way, but right... now we are in a different build universe. I will check that.

Reply:
Removed all *_release DTS overlay files with duplicated configuration. Pity that it does not work that way with .conf files as well.